### PR TITLE
feat: update new procedures naming conventions (MAPCO-4206)

### PR DESCRIPTION
**Pull request by ronenkapelian**
Related to last updates of Shaziri

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         |  ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                      |

https://mapcolonies.atlassian.net/browse/MAPCO-4206

### CODE REVIEW

**Review by asafMasa — APPROVED** [review]

File: src/DAL/migration/customTypes.sql [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 CREATE TYPE "polygon_parts".insert_part_record AS
 (
-	"recordId" uuid,
+	"record_id" uuid,
```
Comment (asafMasa) [inline src/DAL/migration/customTypes.sql:5]: Please update in 'mc-models' all of the column's attributes.
 for example 'imagingTimeBeginUTC' need to be changed to 'imaging_time_begin_UTC' https://github.com/MapColonies/mc-models/blob/master/src/models/polygonParts/polygonPartRecord.ts#L89

File: src/DAL/migration/fullSchema.sql [hunk 2]
```diff
@@ -4,25 +4,25 @@
...
-    "updatedInVersion" text COLLATE pg_catalog."default",
-    "ingestionDateUTC" timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    "imagingTimeBeginUTC" timestamp with time zone NOT NULL,
-    "imagingTimeEndUTC" timestamp with time zone NOT NULL,
-    "resolutionDegree" numeric NOT NULL,
-    "resolutionMeter" numeric NOT NULL,
-    "sourceResolutionMeter" numeric NOT NULL,
-    "horizontalAccuracyCE90" real,
+    "updated_in_version" text COLLATE pg_catalog."default",
+    "ingestion_date_UTC" timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
```
Comment (asafMasa) [inline src/DAL/migration/fullSchema.sql:12]: UTC is upper case, will it work in queries withaout marking "" as it works with lower case by default?

File: src/DAL/migration/local.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+-- call "polygon_parts".insert_part(('24fa4d31-946b-49d2-92c9-89f2ef78bb5e', '123', 'full', '2', '2022-08-22 02:08:10', '2022-08-22 02:08:10', 0.072, 8000, 8000, 4000, 'string', 'world,east,west', 'capitals', 'entire world first version', 'SRID=4326;POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))')::"polygon_parts".insert_part_record);
```
Comment (asafMasa) [inline src/DAL/migration/local.sql:1]: should this file be in this PR?

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good work! 🥇

File: src/DAL/migration/fullSchema.sql (hunk 2 above)
Comment (ronenkapelian) [reply src/DAL/migration/fullSchema.sql:14]: according to docs, upper cases on postgres are sensitive, its better to add "" just for safe

File: src/DAL/migration/customTypes.sql [hunk 4]
```diff
@@ -1,50 +1,25 @@
...
+	"imaging_time_end_UTC" timestamp with time zone,
+	"resolution_degree" numeric,
+	"resolution_meter" numeric,
+	"source_resolution_meter" numeric,
+	"horizontal_accuracy_ce_90" real,
 	sensors text,
-	region text,
-	classification "PolygonParts".classification,
+	countries text,
+    cities text,
```
Comment (asafMasa) [inline src/DAL/migration/customTypes.sql:19]: identation?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/DAL/migration/fullSchema.sql (hunk 2 above)
Comment (ronenkapelian) [reply src/DAL/migration/fullSchema.sql]: done

**Review by asafMasa — APPROVED** [review]

**Pull request merged**